moveit_ros/planning/planning_request_adapter_plugins/src/fix_workspace_bounds.cpp

```diff
@@ -53,13 +54,10 @@ static const rclcpp::Logger LOGGER = rclcpp::get_logger("moveit_ros.fix_workspac
 class FixWorkspaceBounds : public planning_request_adapter::PlanningRequestAdapter
 {
 public:
-  static const std::string WBOUNDS_PARAM_NAME;
-
   void initialize(const rclcpp::Node::SharedPtr& node, const std::string& parameter_namespace) override
   {
-    node_ = node;
-    workspace_extent_ = getParam(node_, LOGGER, parameter_namespace, WBOUNDS_PARAM_NAME, 10.0);
-    workspace_extent_ /= 2.0;
+    param_listener_ =
+        std::make_shared<default_plan_request_adapter_parameters::ParamListener>(node, parameter_namespace);
```

Comment:
nit: `initialize` should take the node by const-value and then `std::move` it here.

Reply:
Not a huge expert on this but I think the main reason why we pass it by reference is

> The advantage of passing the shared_ptr by const& is that the reference count doesn't have to be increased and then decreased. Because these operations have to be thread-safe, they can be expensive.

(Copied from more detailed answer here: https://stackoverflow.com/questions/37610494/passing-const-shared-ptrt-versus-just-shared-ptrt-as-parameter)

Reply:
From the base interface it's not clear if the node is ignored, only used in the function, or transferred. Copying by default doesn't seem like the better option to me.

Reply:
Sounds good